lxa_iobus/server/server.py

```diff
@@ -120,6 +124,13 @@ async def flash_worker(self):
                 if shutdown:
                     return
 
+                self.can_isp.console_log(
+                    'Flashing {} ({})'.format(
+                        node.driver.__class__.__name__,
```

Comment:
When flashing a device the output looks like this:
```
2021-08-19 10:23:58.184756: Flashing EthernetMuxDriver (00000507.00000001.00000004.00000017)
2021-08-19 10:23:58.185115: Invoking isp
2021-08-19 10:23:59.188729: Start flashing
2021-08-19 10:23:59.199363: Writing section flash
2021-08-19 10:24:22.211381: Write 27080 in 23.011596202850342 : 1176.7979831249484 Bytes/sec
2021-08-19 10:24:22.213591: Reseting node
2021-08-19 10:24:22.486137: Flashing done
```
We are flashing an EthernetMux, not an Ethernet-Mux driver here. Also the Address is not that useful to the end-user since it is not written on the device.
What about using the name e.g. `Ethernet-Mux-00003.00023` here?

Reply:
The address was specifically requested by @lichtfeind

@lichtfeind: opinion?

Reply:
What about using the first part of the `name`, keep the address requested by @lichtfeind  and just add the instance number `00003.00023`?

Reply:
I would be fine with the `NAME_PREFIX`(`Ethernet-Mux-00003.`) and the serial number.
I just need a way to see what node is currently getting the update. Otherwise its hard to keep track when flashing all nodes.

If possible it might be nice to see how many nodes are still in the queue.

Reply:
@lichtfeind : I log the name of the device now, which pretty much does that